pkg/unikontainers/unikernels/unikernel.go

```diff
@@ -33,17 +31,19 @@ type UnikernelParams struct {
 	EthDeviceGateway string // The eth device gateway
 	RootFSType       string // The rootfs type of the Unikernel
 	BlockMntPoint    string // The mount point for the block device
+	Version          string // The version of the unikernel
 }
 
 var ErrNotSupportedUnikernel = errors.New("unikernel is not supported")
 
-func New(unikernelType UnikernelType) (Unikernel, error) {
+func New(unikernelType string, unikernelVersion string) (Unikernel, error) {
```

Comment:
Is there a reason that we add the version inside this function, instead of adding it along with the other annotations in the caller?

Reply:
There is no need to. We can move the logic to unikernel.Init() function and set the version provided by unikernelParams. I changed it.